ledger/src/shred.rs

```diff
@@ -556,7 +557,8 @@ pub mod layout {
 
     // Returns slice range of the packet payload which is signed.
     pub(crate) fn get_signed_message_range(packet: &Packet) -> Range<usize> {
```

Comment:
I think this should return an option or result so that the caller is forced to handle the case where the shred size is less than `SIZE_OF_SIGNATURE`

Reply:
This is indeed changed to an `Option` in https://github.com/solana-labs/solana/pull/25631